tests/expose_test.go

```diff
@@ -603,6 +603,36 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			return vmi
 		}
 
+		cleanupVM := func(virtClient kubecli.KubevirtClient, namespace string, vmName string) error {
```

Comment:
Besides tests that actually test deleting the vm. I believe the vm cleanup is not needed and we can use `BeforeTestCleanup`.

Reply:
Just make sure we do delete there everything (jobs, services, etc...)

Reply:
Passing in the vm object itself makes more sense to me (we are deleting a known object).
But the real problem is that you pass a `namespace` and `vmName` but never use them.

Please do not use anonymous functions if it is not a must, everytime I see them I am looking for the reason they exist, i.e. the variable that is shared with the outside).